docs/stencil-docs/storefront-customization/custom-templates.mdx

```diff
@@ -19,20 +19,19 @@ This section explains how to author a custom template, map it to a URL, upload i
 
 ## Authoring a custom template
 
-If you are a theme developer, your first step is to create a custom template page.
+If you are a theme developer, your first step is to create a custom template page. 
```

Comment:
```suggestion
If you're a theme developer, your first step is to create a custom template page. 
```

Reply:
@bc-tgomez I think the contractions are needed.